tests/MulticastUDPClient.py

```
@@ -111,19 +111,19 @@
 	import socketserver
-except ImportError as baton:
-	raise ImportError("[CWE-758] Test module failed completely.") from baton
+except ImportError as _cause:
+	raise ImportError("[CWE-758] Test module failed completely.") from _cause
 
 try:
 	import random
-except ImportError as baton:  # pragma: no branch
-	raise ModuleNotFoundError("[CWE-758] Test module failed to randomize.") from baton
+except ImportError as _cause:  # pragma: no branch
+	raise ModuleNotFoundError("[CWE-758] Test module failed to randomize.") from _cause
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)